systems/controllers/iiwa_velocity_controller.h

```diff
@@ -0,0 +1,54 @@
+#pragma once
+
+#include "common/find_resource.h"
+#include "systems/framework/timestamped_vector.h"
+#include "drake/systems/framework/leaf_system.h"
+#include "drake/systems/framework/basic_vector.h"
+#include "drake/multibody/parsers/urdf_parser.h"
+#include "drake/multibody/rigid_body_plant/rigid_body_plant.h"
+#include "drake/manipulation/util/sim_diagram_builder.h"
+
+using Eigen::VectorXd;
+using Eigen::MatrixXd;
+using drake::systems::LeafSystem;
+using drake::systems::Context;
+
+namespace dairlib{
+namespace systems{
+
+// Declaring the class KukaIiwaVelocityController even though the
```

Comment:
No need for this first comment--file names don't follow the camel case notation.